packages/server-side-render/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "@wordpress/server-side-render",
+	"version": "1.3.0",
```

Comment:
Version is incorrect. It should be anything lower than `1.0.0`: `1.0.0-alpha.1`?